nix/pkgs/go-maven-resolver/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, buildGo116Package, fetchFromGitHub }:
+{ lib, buildGo116Module, fetchFromGitHub }:
```

Comment:
Oh, nice catch. I must have broken that last time when we were upgrading Go verison.